tests/jenkins/TestUploadToS3.groovy

```diff
@@ -0,0 +1,20 @@
+import jenkins.tests.BuildPipelineTest
+import org.junit.Before
+import org.junit.Test
+
+
+class TestUploadToS3 extends BuildPipelineTest {
+
+    @Before
+    void setUp() {
+
+        this.registerLibTester(new TestUploadToS3LibTester( '/tmp/src/path', 'dummy_bucket', '/upload/path' ))
+
+        super.setUp()
+    }
+
+    @Test
+    void testUploadToS3() {
```

Comment:
Does this need to be `public void` method? Otherwise I'm afraid this might get any exception like `groovy.lang.missingpropertyexception`
Same concern for all other Test methods.

Reply:
It's not mandatory. default is public